documentation/modules/security/proc-accessing-kafka-using-ingress.adoc

```diff
@@ -60,30 +77,85 @@ spec:
             host: broker-1.myingress.com
           - broker: 2
             host: broker-2.myingress.com
+          class: nginx  
     # ...
   zookeeper:
     # ...
 ----
-<1> Ingress hosts for the bootstrap service and Kafka brokers.
++
+Add any hostname to the `bootstrap` and `broker-<index>` prefixes that identify the bootstrap and brokers.
++
+NOTE: You might need to add `nginx` as the class if it is missing in the ingresses created. 
 
 . Create or update the resource.
 +
 [source,shell,subs=+quotes]
+----
 kubectl apply -f _<kafka_configuration_file>_
+----
++
+A cluster CA certificate to verify the identity of the kafka brokers is created in the secret `my-cluster-cluster-ca-cert`.
++
+`ClusterIP` type services are created for each Kafka broker, as well as an external bootstrap service.
++
+An `ingress` is also created for each service, with a DNS address to expose them using the NGINX Ingress Controller.
++
+.Ingresses created for the bootstrap and brokers
+[source,shell]
+----
+NAME                        CLASS  HOSTS                    ADDRESS               PORTS
+my-cluster-kafka-0          nginx  broker-0.myingress.com   external.ingress.com  80,443
+my-cluster-kafka-1          nginx  broker-1.myingress.com   external.ingress.com  80,443
+my-cluster-kafka-2          nginx  broker-2.myingress.com   external.ingress.com  80,443
+my-cluster-kafka-bootstrap  nginx  bootstrap.myingress.com  external.ingress.com  80,443
+----
++
+The DNS addresses used for client connection are propagated to the `status` of each ingress.
++
+.Status for the bootstrap ingress
+[source,yaml]
+----
+status:
+  loadBalancer:
+    ingress:
+      - hostname: external.ingress.com
+ # ...
+----
+
+. Use a target broker to check the client-server TLS connection on port 443 using the OpenSSL `s_client`.  
++
+[source,shell]
+----
+openssl s_client -connect my-cluster-kafka-0:443 -servername my-cluster-kafka-0 -showcerts
```

Comment:
Should this use the example addresses from above? E.g. `broker-0.myingress.com` instead of `my-cluster-kafka-0`?